test/TestAsset/http-status-codes.xml

```diff
@@ -0,0 +1,368 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
@Ocramius 

Changing the file name by URL in ResponseTest.php:77,  works just fine.

However, this can be a problem.

The test will depend on an external online resource.

And besides this, imagine if all libraries decide to do the same thing, requesting the file when the test runs.

I believe this may be end up leading to a case of service misuse.

One solution would be to add a pre-release script, something like this in composer.json ("scripts" block).

   "pre-release-tasks": [
      "@update-test-asset"
    ],
    "update-test-asset": "curl -o test/TestAsset/http-status-codes.xml https://www.iana.org/assignments/http-status-codes/http-status-codes.xml"

What do you think?